src/ai/ai.cpp

```diff
@@ -19,7 +19,7 @@ float estimate_defensive_strength(sys::state& state, dcon::nation_id n) {
 
 		auto other = dr.get_related_nations(0) != n ? dr.get_related_nations(0) : dr.get_related_nations(1);
 		if(other.get_overlord_as_subject().get_ruler() != n)
-			value += estimate_strength(state, other);
+			value += other.get_military_score();
```

Comment:
Why are we not calling estimate strength here?

Reply:
the estimate_strength function also addup the overlords themselves, so when an overlord != n then it will duplicate by the subjects